.github/labeler.yml

```diff
@@ -1,21 +1,30 @@
 area/frontend:
   - airbyte-webapp/*
+  - airbyte-webapp/**/*
 
 area/core:
   - airbyte-server/*
+  - airbyte-server/**/*
   - airbyte-scheduler/*
+  - airbyte-scheduler/**/*
 
 area/protocol:
   - airbyte-protocol/*
+  - airbyte-protocol/**/*
 
 area/worker:
   - airbyte-workers/*
+  - airbyte-workers/**/*
 
 area/connectors:
-  - airbyte-integrations/connectors/*
+  - airbyte-integrations/bases/**/*
```

Comment:
base-normalization is going to be labeled as connector?

Reply:
I added a pattern !base-normalization to exclude normalization from `area/connectors` label

Reply:
should there be an automatic label for normalization though?

Reply:
added